### ISSUE TITLE
Please allow different media types

### ISSUE THREAD

**Issue by rahulanand16nov** [engineerd/wasm-to-oci#27]
If I am understanding this correctly, `wasm-to-oci` uses `application/vnd.wasm.content.layer.v1+wasm` media type. In the official artifact layer type, we can see this media type: https://github.com/opencontainers/artifacts/blob/main/artifact-authors.md#example-layer-types.

Is there any future plans to allow different media types?

**Comment by radu-matei** [comment]
I agree that the media type should be configurable -- do you have any suggestions for a better default?
Thanks!

**Comment by rahulanand16nov** [comment]
@radu-matei Sorry for the late reply. These modules are in large have been used as extensions for the edge compute (like proxies). For example, Envoy doesn't accept the current default media type but instead uses `application/vnd.oci.image.layer.v1.tar+gzip`. Having a default media type that covers a large part of wasm's use case would be better IMO. Let me know what you think.
